test/test_FiniteDifferences2D.jl

```diff
@@ -46,10 +46,16 @@ end
                 @parallel inn!(R, Axxyy) = (@all(R) = @inn(Axxyy); return)
                 @parallel inn_x!(R, Axx) = (@all(R) = @inn_x(Axx); return)
                 @parallel inn_y!(R, Ayy) = (@all(R) = @inn_y(Ayy); return)
+                @parallel idx_x!(R, A)   = (@all(R) = @idx_x(A); return)
```

Comment:
This line with `idx_` as well as all the following does not belong in this PR, does it?